core/src/test/java/nl/nn/adapterframework/doc/testtarget/groups/Pipe.java

```diff
@@ -0,0 +1,9 @@
+package nl.nn.adapterframework.doc.testtarget.groups;
+
+import nl.nn.adapterframework.doc.IbisDoc;
+
+public class Pipe implements IPipe {
+	@IbisDoc("10")
+	public void setListener(Listener listener) {
+	}
+}
```

Comment:
Het is heel goed dat je test classes en interfaces maakt, maar ik kan me voorstellen dat het verwarrend gaat zijn dat je daarvoor bekende termen als 'Pipe' en 'Listener' gebruikt, met name als ze af gaan wijken van wat bekend is.
In deze class heeft een 'Pipe' een 'Listener'. Dat is in het echt niet zo.
Ik herinner me nog een telefoongesprek waarin jij zoiets zei als dat een Pipe een Listener had. Je bedoelde toen misschien dit geval, maar ik was toen in verwarring.
Ik denk dat je hier dus beter andere termen kan gebruiken, zoals Parent en Child, of StructuredItem en SubItem

Reply:
Ik heb dit punt gedeeltelijk opgelost. Op deze specifieke plaats heb ik de namen veranderd, maar ik heb ook ergens de naam 
`IListener` laten staan. Dat is in package `nl.nn.adapterframework.doc.testtarget.simple`. Er is daar geen conflict omdat alleen het woord "Listener" overeen komt met het FF!. Alle andere classes in dit package hebben namen die niets met het FF! te maken hebben. Een andere reden om dit tweede geval niet te veranderen is dat de gerelateerde test code complexer is dus moeilijker aan te passen. Maar als ik er een of twee uur aan besteed dan is het gebeurd. Wil je dat ik dit af maak of is het goed zoals ik het gemaakt heb?

Reply:
Ik heb geen bezwaar gezien. Daarom resolve ik hem nu.